### ISSUE TITLE
Missing name in `helm install` in TL;DR

### ISSUE THREAD

**Issue by lindhe** [rowanruseler/helm-charts#244]
The release name is missing here:

https://github.com/rowanruseler/helm-charts/blob/87ecfd8dba3059bc07653f3df13c6b2b537e6dc1/charts/pgadmin4/README.md?plain=1#L11

Add either `--generate-name` or a name like so: `helm install pgadmin runix/pgadmin4`.

🙃

**Comment by rowanruseler** [comment]
You are correct, feel free to open a pull request! Else I'll do it next week :)

**Comment by rowanruseler** [comment]
added as requested: https://github.com/rowanruseler/helm-charts/pull/250
